client/src/components/atoms/SearchableDropdown.tsx

```diff
@@ -1,24 +1,29 @@
-import React, { FunctionComponent, useState } from "react";
+import React, { FunctionComponent, useEffect, useState } from "react";
```

Comment:
ignore this file for now
i had to make tons of modifcations, but most edits will be reverted once the SearchableTagInput component is done